openapi/products/listingBuilder.yaml

```diff
@@ -0,0 +1,214 @@
+          description: Return listings for the specified business entity
+          required: true
+        - schema:
+            type: string
+          in: header
+          name: Accept-Language
+      responses:
+        '200':
+          description: OK
+          content:
+            application/vnd.api+json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/listListings'
+                  links:
+                    type: object
+                    properties:
+                      self:
+                        type: string
+                        format: uri
+                      first:
+                        type: string
+                        description: Provides a link back to the first page of results
+                        format: uri
+                      next:
+                        type: string
+                        description: The URI at which the next batch of listings can be gotten from
+                        format: uri
+    options:
+      operationId: options-businessEntityListings
+      summary: List valid HTTP verbs for /businessEntityListings
+      description: 'Used solely for [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) the OPTIONS request returns the list of possible HTTP methods and other headers that browsers use to protect user''s security. You should not call this operation directly. '
+      responses:
+        '204':
+          description: No Content
+        '403':
+          description: User Cannot Access Entity
+        '404':
+          description: Business Not Found
+        '406':
+          description: No Listing Product Active
+      tags:
+        - Options
+        - Business Entity Listings
+    parameters: []
+components:
+  schemas:
+    businessLocationReviews:
+      title: Business Entity Listings
+      type: object
+      x-lifecycle:
+        status: trustedTester
+      description: |-
+        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Trusted Tester`
+        A review of an item - for example, of a restaurant, movie, or store.
+      x-tags:
+        - Business Entity Listings
+      properties:
+        name:
+          description: Public facing name of the source
+          type: string
+          example: HotFrog
+        icon:
+          description: Link to icon provided by Vendasta’s media service
+          type: string
+          format: uri
+          example: //www.cdnstyles.com/static/images/icon32/sourceId-10090.png
+        popularity:
+          description: Popularity of the source relative to other sources
+          type: number
+        sourceTypeCode:
+          description: Type of source, business directory, app (Snapchat), GPS app, etc.
+          type: string
+          example: BUSINESS_DIRECTORY
```

Comment:
Enum values should be camelCase. Also please document the enum options and include a text description.